client/src/components/SitterProfile/SitterProfile.tsx

```diff
@@ -0,0 +1,97 @@
+import {
+  Typography,
+  Grid,
+  CircularProgress,
+  Paper,
+  Card,
+  CardMedia,
+  CardContent,
+  GridList,
+  GridListTile,
+} from '@material-ui/core';
+import { useHistory } from 'react-router-dom';
+import { useAuth } from '../../context/useAuthContext';
+import house from '../../Images/house.jpg';
+import AvatarDisplay from '../AvatarDisplay/AvatarDisplay';
+import LocationOnIcon from '@material-ui/icons/LocationOn';
+import useStyles from './useStyles';
+import cuteDogOne from '../../Images/dog-puppy-on-garden-royalty-free-image-1586966191.jpg';
+import cuteDogTwo from '../../Images/dog-unsolicited.jpg';
+import cuteDogThree from '../../Images/siberian_husky_cute_puppies.jpg';
+
+const SitterProfile = (): JSX.Element => {
+  const classes = useStyles();
+  const { loggedInUser } = useAuth();
+  const history = useHistory();
+
+  if (loggedInUser === undefined) return <CircularProgress />;
+  if (!loggedInUser) {
+    history.push('/login');
+    return <CircularProgress />;
+  }
+
+  const tileData = [
+    {
+      img: cuteDogOne,
+      title: 'Dog',
+    },
+    {
+      img: cuteDogTwo,
+      title: 'Another dog',
+    },
+    {
+      img: cuteDogThree,
+      title: 'Even more dog',
+    },
+  ];
+
+  return (
+    <Card className={classes.mainCard} elevation={5}>
+      <CardMedia className={classes.cardMedia} image={house} />
+      <Paper className={classes.mainPaper} elevation={3}>
```

Comment:
Ditto as above.

Reply:
Removed the Paper element with no issues.